src/p2f/src/CodeOwls.PowerShell/CodeOwls.PowerShell.Provider/Provider.cs

```diff
@@ -54,6 +56,34 @@ public abstract class Provider : NavigationCmdletProvider,
         IContentCmdletProvider
     {
         private static readonly Dictionary<string, Regex> FilterRegexMap = new Dictionary<string, Regex>(StringComparer.OrdinalIgnoreCase);
+        private static readonly string StringSeperator = (IsWindows) ? "\\" : "/";
+        private static readonly char CharSeperator = (IsWindows) ? '\\' : '/';
```

Comment:
Separator (typo in spelling) - applies to all instances in this file

Reply:
good catch. Thanks Raghu.